docs/source/conf.py

```diff
@@ -10,19 +10,22 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+import os
+import sys
+
+sys.path.insert(
+    0, os.path.abspath("\home\kiran\housingproject\src\housingpackage")
```

Comment:
please make this as a relative path (Ex: https://github.com/Comfortably-Numb3/mle-training/blob/c0075d0194b77810dd7c0aa85dea837db9732261/docs/source/conf.py#L17). Otherwise, others will get errors when they try to create `docs`

Reply:
Done